test/smokes/flake8/expectations.rb

```diff
@@ -231,25 +240,41 @@
   analyzer: :_
 )
 
-s.add_test("dot_python_version", type: "success", issues: [], analyzer: { name: "Flake8", version: default_version })
+s.add_test(
+  "dot_python_version",
+  type: "success",
+  warnings: [
+    { message: /Sider's recommended configuration file is about to be released in early Mar 2021.\nAfter the release, Sider will automatically apply our recommended ruleset if you don't have the Flake8 configuration file in your repository./, file: nil }
+  ],
+  issues: [], analyzer: { name: "Flake8", version: default_version }
```

Comment:
Please insert new-line between issues and analyzer.

```suggestion
  issues: [],
  analyzer: { name: "Flake8", version: default_version }
```

Reply:
Done.